src/languages/en.ts

```diff
@@ -3670,6 +3670,34 @@ export default {
             whatsMainReason: "What's the main reason you're disabling auto-renew?",
             renewsOn: ({date}) => `Renews on ${date}.`,
         },
+        requestEarlyCancellation: {
+            title: 'Request early cancellation',
+            subtitle: 'What’s the main reason you’re requesting early cancellation?',
+            subscriptionCancelled: {
```

Comment:
```suggestion
            subscriptionCanceled: {
```

Easy to find the key/text if the spelling matches.